### PR TITLE
Update wsl page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates `/wsl` as per the [copydoc](https://docs.google.com/document/d/1M7e2B08GgPA7K6Wx0ThE6s9_xQINM0MSkarfdKUTEYI/edit) and [design](https://user-images.githubusercontent.com/9105999/157416010-031da986-9b50-412a-ba65-ae502ef73aa5.png) more info can be found in the issue.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11341.demos.haus/wsl    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against design and copydoc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4838

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @petesfrench, looks good 👍

File: templates/wsl/index.html [hunk 1]
```diff
@@ -18,9 +18,8 @@ <h1>Ubuntu on WSL</h1>
       <p>Install a complete Ubuntu terminal environment in minutes on Windows 10 with Windows Subsystem for Linux (WSL).</p>
       <p>Access the Linux terminal on Windows, develop cross-platform applications, and manage IT infrastructure without leaving Windows.</p>
       <a href="https://www.microsoft.com/store/productId/9NBLGGH4MSV6" class="p-link--external p-button--positive">Download from the Microsoft Store</a>
```
Comment (minkyngkm) [inline templates/wsl/index.html:20]:
```suggestion
      <a href="https://www.microsoft.com/store/productId/9NBLGGH4MSV6" class="p-button--positive">Download from the Microsoft Store</a>
```

File: templates/wsl/index.html [hunk 2]
```diff
@@ -134,163 +111,137 @@ <h3 class="p-matrix__title p-heading--4">Kubernetes</h3>
...
+        width="152",
+        height="64",
+        hi_def=True,
+        loading="lazy",
+        attrs={"style": "padding-bottom:1rem;padding-top:16px;"}
+        ) | safe
+      }}
+      <h4>Cross Platform Development</h4>
+      <p>Create and test your CI/CD pipelines locally on an Ubuntu WSL instance. When ready, publish to a cloud production environment running Ubuntu VMs.</p>
+      <p><a class="p-link--external p-link--inverted" href="https://youtu.be/DmfuJzX6vJQ">Discover how to easily access “Kubernetes on Windows with WSL2” in our webinar</a></p>
```
Comment (minkyngkm) [inline templates/wsl/index.html:196]:
```suggestion
      <p><a class="p-link--inverted" href="https://youtu.be/DmfuJzX6vJQ">Discover how to easily access “Kubernetes on Windows with WSL2” in our webinar</a></p>
```

File: templates/wsl/index.html [hunk 3]
```diff
@@ -134,163 +111,137 @@ <h3 class="p-matrix__title p-heading--4">Kubernetes</h3>
...
+        ) | safe
+      }}
+      <h4>Enterprise support</h4>
+      <p>Ubuntu is certified on WSL through close collaboration with Microsoft. Enterprise support is provided for Ubuntu from Azure to Windows workstations creating a seamless operating environment.</p>
+      <p><a class="p-button js-invoke-modal" data-testid="interactive-form-link" class="p-button" href="https://ubuntu.com/wsl/contact-us">Get in touch</a></p>
+    </div>            
+  </div>  
 </section>
 
 <section class="p-strip is-deep">
```
Comment (minkyngkm) [inline templates/wsl/index.html:247]: This section "Getting Started with WSL" doesn't seem to be updated yet?

**Review by minkyngkm — CHANGES_REQUESTED** [review]
Mostly looks good! I just found some minor things to be updated!
- The `p-link--external` classes haven't been removed 
- The tutorials in the "Getting Started with WSL" section 
- ["View more from our blog"](http://ubuntu.com/blog/tag/wsl) is missing in the last section.

File: templates/wsl/index.html (hunk 3 above)
Comment (petesfrench) [reply templates/wsl/index.html:247]: This is being done in another PR here: https://github.com/canonical-web-and-design/ubuntu.com/pull/11346

**Review by minkyngkm — APPROVED** [review]
LGTM Pete! 👍

**Pull request merged**